nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/pom.xml

```diff
@@ -147,6 +147,12 @@
             <version>2.2.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-solr-client-api</artifactId>
+            <version>1.8.0-SNAPSHOT</version>
+            <scope>provided</scope>
```

Comment:
The solr-bundle NAR pom also needs to be updated to have a parent of the client API NAR.

Also, the assembly pom needs to include the client API NAR so it ends up in the final build.

Reply:
> The solr-bundle NAR pom also needs to be updated to have a parent of the client API NAR.

Could you explain? I added the client api nar to the assemble, rebuilt and was able to bring in a new client service just fine.

Reply:
I think we're good here.